### ISSUE TITLE
seriesname - is it required and/or how to get around using it?

### ISSUE THREAD

**Issue by gscoon** [node-influx/node-influx#149]
I'm new to influxdb. I'm trying to figure out if and why I need a series name to call the writePoint function.  My understanding is that series are defined as unique sets of measurement and tags.  I realize that there were significant version changes recently, but am I missing something?

**Comment by connor4312** [comment]
Hi gscoon,

Each set of measurements is identified by a series name. For more info you can check out Influx's "Writing and exploring data" tutorial [here](https://docs.influxdata.com/influxdb/v0.13/introduction/getting_started/#writing-and-exploring-data). Let us know if you have any issues!

**Comment by gscoon** [comment]
Thanks for the really quick response.  I've checked out that link and others, but there's no mention of "series name."  Maybe it's just a semantics issue, but when is a series name not the same as a measurement?

**Comment by connor4312** [comment]
Technically it's a bit of a misnomer here; the "seriesName" _is_ actually the measurement name. In Influx each series is a set of records with a unique set of tags. For instance, `insert foo,bar=a; insert foo,bar=b` would create the series `foo,bar=a` and `foo,bar=b`. Both of these are grouped in the measurement `foo`. Influx has a good page on [key concepts](https://docs.influxdata.com/influxdb/v0.13/concepts/key_concepts/#measurement) about this stuff.

tl;dr in this library `measurement` is incorrectly referred to as `seriesName`. I'll reopen this and put it on my list to fix in 5.0.0 😄

**Comment by gscoon** [comment]
Thanks for clearing things up Connor.

**Comment by thinkdevcode** [comment]
This was the major breaking change I did in my fork of this library - series are now something "behind-the-scenes" that the db handles and not something we write/manage from our end (as of 0.9). This is also the reason why I never made a PR with my changes as I figured it would be too major.

**Comment by MusicalCreeper01** [comment]
This also confused me yesterday, being a beginner with InfluxDB. It might be worth it to add a note in the readme that the series name is also the measurement name, so it's not confusing for beginners and first-time users.

**Comment by connor4312** [comment]
Resolved with https://github.com/node-influx/node-influx/pull/205

**Comment by connor4312** [comment]
These changes have been published in `5.0.0-alpha.1`. You can install this via:

```
npm install --save influx@next
```
